delinea_privilege_manager/README.md

```diff
@@ -0,0 +1,178 @@
+
+| Template     | Event Name    | Name |
+| ---------  | -------------- |--------------
+| Send Application Action Events to Syslog | Application Action Events | Application Action Events |
+| Send Application Justification Events to Syslog | Application Justification Events | Application Justification Events |
+| Send Change History Events to Syslog | Not Applicable | Change History Events |
+| Send Newly Discovered File Events to Syslog | Newly Discovered File Events | Newly Discovered File Events |
+| Send Password Disclosure Events to Syslog | Password Disclosure Events | Password Disclosure Events |
+| Send Bad Rated Application Action Events to Syslog | Bad Rated Application Action Events | Bad Rated Application Action Events |
+
+### Validation
+
+[Run the Agent's status subcommand][5] and look for `Delinea Privilege Manager` under the Checks section.
+
+## Data Collected
+
+### Log 
+
+| Format     | Event Types    |
+| ---------  | -------------- |
+| CEF | Application Action Events, Bad Rated Application Action Events, Application Justification Events, Password Disclosure Events, Newly Discovered File Events, Change History Events |
+
+### Metrics
+
+The Delinea Privilege Manager integration does not include any metrics.
+
+### Events
+
+The Delinea Privilege Manager integration does not include any events.
+
+### Service Checks
+
+The Delinea Privilege Manager integration does not include any service checks.
+
+## Troubleshooting
+
+**Permission denied while port binding:**
+
+If you see a **Permission denied** error while port binding in the Agent logs:
+
+1. Binding to a port number under 1024 requires elevated permissions. Grant access to the port using the `setcap` command:
+    ```shell
+    sudo setcap CAP_NET_BIND_SERVICE=+ep /opt/datadog-agent/bin/agent/agent
+    ```
+
+2. Verify the setup is correct by running the `getcap` command:
+
+    ```shell
+    sudo getcap /opt/datadog-agent/bin/agent/agent
+    ```
+
+    With the expected output:
+
+    ```shell
+    /opt/datadog-agent/bin/agent/agent = cap_net_bind_service+ep
+    ```
+
+    **Note**: Re-run this `setcap` command every time you upgrade the Agent.
+
+3. [Restart the Agent][2].
+
+
+**Data is not being collected:**
+
+Ensure traffic is bypassed from the configured port if the firewall is enabled.
+
+**Port already in use:**
+
+If you see the **Port <PORT_NUMBER> Already in Use** error, see the following instructions. The following example is for port 514:
+
+- On systems using Syslog, if the Agent listens for events on port 514, the following error can appear in the Agent logs: `Can't start UDP forwarder on port 514: listen udp :514: bind: address already in use`. This error occurs because by default, Syslog listens on port 514. To resolve this error, take **one** of the following steps: 
```

Comment:
```suggestion
- On systems using Syslog, if the Agent listens for events on port 514, the following error can appear in the Agent logs: `Can't start UDP forwarder on port 514: listen udp :514: bind: address already in use`. This error occurs because, by default, Syslog listens on port 514. To resolve this error, take **one** of the following steps: 
```

Reply:
Have updated as per suggestion.